### ISSUE TITLE
URITests failure test_energi-qt

### ISSUE THREAD

**Issue by RyanLucchese** [energicryptocurrency/energi#146]
The `test_energi-qt` binary has a test failure in the URITests related to checking a Dash address. This will need to be replaced with an Energi address so the test passes.

```
$ energi/bin/test_energi-qt 
********* Start testing of URITests *********
Config: Using QtTest library 5.5.1, Qt 5.5.1 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 5.4.0 20160609)
PASS   : URITests::initTestCase()
FAIL!  : URITests::uriTests() 'rv.address == QString("XwnLY9Tf7Zsef8gMGL2fhWA9ZmMjt4KPwg")' returned FALSE. ()
   Loc: [qt/test/uritests.cpp(55)]
PASS   : URITests::cleanupTestCase()
Totals: 2 passed, 1 failed, 0 skipped, 0 blacklisted
********* Finished testing of URITests *********
```

**Comment by RyanLucchese** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [URITests failure test_energi-qt](https://trello.com/c/gHkEV0Vv/94-uritests-failure-testenergi-qt)

**Comment by RyanLucchese** [comment]
Fixed by #179
